src/main/app/cypress/data/koulutus.ts

```diff
@@ -89,13 +89,25 @@ const getLukioFields = ({ tyyppi }) => {
   });
 };
 
+const getTuvaFields = ({ tyyppi }) => {
+  return _.merge(getBaseFields({ tyyppi }), {
```

Comment:
Vaihdoin aiemmin nähin cypressin filuihin, että käytetään suoraan lodash/fp:stä tuota mergeä ja niitä paria muuta funktiota. On vähän hämmentävää, että tämä ilmeisesti jostain syystä toimii nyt näinkin, vaikka importti puuttuu kokonaan tuolle alaviivalle 😅 .
```suggestion
  return merge(getBaseFields({ tyyppi }), {
```